src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Field.java

```diff
@@ -104,6 +107,14 @@ public String getChildType() {
         return this.children.childType;
     }
 
+    public void setOption(SQLAggregateOption option) {
```

Comment:
consider renaming to setAggregateOption, because setOption doesn't tell much about content

Reply:
ok will do